lebedigital/BeamDesign/BeamDesign.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 16 11:13:33 2022
+
+@author: zv1174
+"""
+import math
+
+def section_dimension_rule_of_thumb(span:float)->tuple:
+    """
+    This is as per eurocode guideline choice of section dimension based on span 
+
+    Parameters
+    ----------
+    span : float
+        beam span.
+
+    Returns
+    -------
+    tuple
+        tuple of breath and depth of the beam.
+
+    """
+    #span/depth ratio for simply supported beam is 15 Therefore,
+    d = 50*round((span/15)/50)
+    #width of beam
+    b = 0.5*d
+    return b,d 
+
+def max_bending_moment_and_shear_force(span:float,load:float,load_type:str) ->tuple:
+    """
+    function to compute max bending moment and shear force for simply supported beam with point load or distributed load
+
+    Parameters
+    ----------
+    span : float
+        beam span.
+    load : float
+        beam load.
+    type : str
```

Comment:
why don't you specify both a point load (add documentation that this is in the center) and a distributed load. If one is intereste in one or the other, they could just set the corresponding values. What are the units?

Reply:
i thought this is much a easier and one can then do superposition  on their own in the script which one has to execute for optimization not that this is just functions to support execution and execution script has to be built using those function. This is how i understood one codes. If not then we have to define a problem and then we can make specific execution script

Reply:
I think including the superposition within this function is much more user friendly.